package.json

```diff
@@ -30,17 +30,15 @@
   "devDependencies": {
     "@commitlint/cli": "^16.2.1",
     "@commitlint/config-conventional": "^16.2.1",
-    "eslint": "^8.10.0",
+    "eslint": "^9.17.0",
     "husky": "^7.0.4",
     "lint-staged": "^12.3.5",
     "mocha": "^9.2.1",
     "prettier": "^2.5.1"
   },
   "peerDependencies": {
-    "eslint": "^7 || ^8"
-  },
-  "dependencies": {
-    "eslint-plugin-compat": "^4.0.2"
```

Comment:
This was mostly in place to handle IE11. However, IE11 support has since been dropped by Salesforce.